src/gui/transferlistfilterswidget.cpp

```diff
@@ -269,7 +269,9 @@ void StatusFilterWidget::updateTorrentStatus(const BitTorrent::Torrent *torrent)
     const bool isStalledDownloading = (torrent->state() ==  BitTorrent::TorrentState::StalledDownloading);
     update(TorrentFilter::StalledDownloading, isStalledDownloading, m_nbStalledDownloading);
 
-    update(TorrentFilter::Checking, torrent->isChecking(), m_nbChecking);
+    const bool isActiveChecking = (!torrent->isErrored() && torrent->isChecking());
```

Comment:
I would switch the two conditions: `torrent->isChecking() && !torrent->isErrored()`

Reply:
Doesn't "active" confuse? They can be queued or even stopped. 
The names of states from libtorrent are directly used here, but they can be confusing. For example, `checking` means `isn't checked yet`, `downloading` means `checked but isn't downloaded yet`.

Reply:
Changed to isChecking.